NotificationService/NotificationService.SvCommon/Common/StartupCommon.cs

```diff
@@ -157,7 +157,7 @@ public void ConfigureServicesCommon(IServiceCollection services)
 
             _ = services.AddAuthentication(ApplicationConstants.BearerAuthenticationScheme).AddJwtBearer(options =>
             {
-                options.Authority = this.Configuration[ConfigConstants.AuthorityConfigKey];
```

Comment:
there is some dependency with auth v2 for this URL?

Any reason why did you changed this?

Reply:
this is changed for V2 related changes, It will work with bot v1 and v2, but with old one it works only with v1, with v2 it will fail